src/tests/suite/daos_rebuild.c

```diff
@@ -39,7 +39,7 @@
 #define OBJ_CLS		DAOS_OC_R3S_RW
 #define OBJ_REPLICAS	3
 #define DEFAULT_FAIL_TGT 0
-
+#define REBUILD_POOL_SIZE	(4ULL << 28)
```

Comment:
just curious that above should be same as (1ULL << 30).

Reply:
indeed